build-conflicts.php

```diff
@@ -232,16 +232,23 @@ function () use ($execute) {
     };
 
     $commitComposerJson = function (string $composerJsonPath) use ($runInPath, $execute) : void {
+        $originalCommit = $runInPath(
+            function () use ($execute) {
+               return $execute('git log -1 --oneline --no-merges');
+            },
+            dirname($composerJsonPath) . '/../security-advisories'
+        );
+
         $runInPath(
-            function () use ($composerJsonPath, $execute) {
+            function () use ($composerJsonPath, $originalCommit, $execute) {
                 $execute('git add ' . escapeshellarg(realpath($composerJsonPath)));
 
                 $message = sprintf(
                     'Committing generated "composer.json" file as per "%s"',
                     (new DateTime('now', new DateTimeZone('UTC')))->format(DateTime::W3C)
                 );
 
-                $execute('git diff-index --quiet HEAD || git commit -m ' . escapeshellarg($message));
+                $execute('git diff-index --quiet HEAD || git commit -m ' . escapeshellarg($message) . PHP_EOL . $originalCommit);
```

Comment:
Use `\n` here: using cross-platform newlines leads to inconsistent output if run on other machines

Reply:
The original commit must also run through `escapeshellarg`. Note that `escapeshellarg` adds quotes, so put the concatenation within the existing call